pkg/webui/styles/utilities/general.styl

```diff
@@ -367,43 +367,43 @@
 
   // DISPLAY UTILITIES.
   .d-flex
-    display: flex
+    display: flex !important
```

Comment:
Been looking at the whole `!important` issue again, and it appears to be somewhat consensus that `!important` on utility classes is OK.
https://css-tricks.com/when-using-important-is-the-right-choice/

But let's make sure we add an ignore comment for the linter then.